File: CloudBridge-PM/src/CloudBridge/network/protocol/packet/ConnectionPacket.php
<?php

namespace CloudBridge\network\protocol\packet;

class ConnectionPacket extends Packet {

    public string $server = "";

    public function encode(): void {
        parent::encode();
        $this->put($this->server);
    }

    public function decode(): void {
        parent::decode();
        $this->server = $this->get();
    }

    public function getId(): int {
        return self::ID_CONNECTION;
    }

    public static function create(string $server): self {
        $pk = new self;
        $pk->server = $server;
        return $pk;
    }
}

File: CloudBridge-WD/src/main/java/de/r3pt1s/CloudBridge/network/protocol/packet/ConnectionPacket.java
// This class was created by r3pt1s
package de.r3pt1s.CloudBridge.network.protocol.packet;

public class ConnectionPacket extends Packet {

    public Object server = "";

    @Override
    public void encode() {
        super.encode();
        put(server);
    }

    @Override
    public void decode() {
        super.decode();
        server = get();
    }

    @Override
    public Integer getId() {
        return ID_CONNECT;
    }

    public static ConnectionPacket create(String server) {
        ConnectionPacket pk = new ConnectionPacket();
        pk.server = server;
        return pk;
    }
}
